### PR TITLE
Move .NET SDK CI article from tools, under DevOps. Update accordingly.

### LINKED ISSUE #27722: CI with .NET SDK article looks outdated
Haven't fully read through it, but it references .NET Core 1.0.1. So it's too outdated. I also think it might make sense to move the article from `core/tools` to `devops`

cc @IEvangelist 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8cc010c2-97fa-cf36-689e-c328863397bf
* Version Independent ID: 5d3c5144-af16-1b28-fd84-1457c534037b
* Content: [Continuous Integration (CI) with .NET SDK and tools - .NET CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/using-ci-with-cli)
* Content Source: [docs/core/tools/using-ci-with-cli.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/using-ci-with-cli.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Move CI article from tools, under DevOps. Update accordingly.

Fixes #27722

### CODE REVIEW

File: docs/devops/dotnet-cli-and-continuous-integration.md [hunk 1]
```diff
@@ -1,45 +1,52 @@
...
 
-This document outlines using the .NET SDK and its tools on a build server. The .NET toolset works both interactively, where a developer types commands at a command prompt, and automatically, where a Continuous Integration (CI) server runs a build script. The commands, options, inputs, and outputs are the same, and the only things you supply are a way to acquire the tooling and a system to build your app. This document focuses on scenarios of tool acquisition for CI with recommendations on how to design and structure your build scripts.
+# Use the .NET SDK with Continuous Integration (CI)
+
+This document outlines using the .NET SDK and its tools on a build server. The .NET toolset works both interactively, where a developer types commands at a command prompt, or automatically, where a Continuous Integration (CI) server runs a build script. The commands, options, inputs, and outputs are the same, and the only things you supply are a way to acquire the tooling and a system to build your app. This document focuses on scenarios of tool acquisition for CI with recommendations on how to design and structure your build scripts.
```
Comment (tdykstra) [inline docs/devops/dotnet-cli-and-continuous-integration.md:9]:
```suggestion
This article outlines how to use the .NET SDK and its tools on a build server. The .NET toolset works both interactively, where a developer types commands at a command prompt, and automatically, where a Continuous Integration (CI) server runs a build script. The commands, options, inputs, and outputs are the same, and the only things you supply are a way to acquire the tooling and a system to build your app. This article focuses on scenarios of tool acquisition for CI with recommendations on how to design and structure your build scripts.
```

File: docs/fundamentals/toc.yml [hunk 2]
```diff
@@ -1985,6 +1985,8 @@ items:
   - name: GitHub Actions and .NET
     href: ../devops/github-actions-overview.md
     displayName: "continuous integration,continuous deployment,continuous delivery,ci,cd,ci/cd,github,github action,github actions,lifecycle"
+  - name: .NET CLI and Continuous Integration
+    href: ../devops/dotnet-cli-and-continuous-integration.md
```
Comment (tdykstra) [inline docs/fundamentals/toc.yml:1989]: And remove it from lines 380-381

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of minor comments

File: docs/devops/dotnet-cli-and-continuous-integration.md [hunk 3]
```diff
@@ -1,45 +1,52 @@
 ---
-title: Continuous Integration (CI) with .NET SDK and tools
+title: Use the .NET SDK with Continuous Integration (CI)
```
Comment (IEvangelist) [inline docs/devops/dotnet-cli-and-continuous-integration.md:2]:
```suggestion
title: Use the .NET SDK in Continuous Integration (CI) environments
```

File: docs/devops/dotnet-cli-and-continuous-integration.md [hunk 4]
```diff
@@ -1,45 +1,52 @@
...
 description: Learn how to use the .NET SDK and its tools on the build server with continuous integration.
-ms.date: 05/18/2017
+ms.date: 07/13/2022
 ---
-# Using the .NET SDK and tools in Continuous Integration (CI)
 
-This document outlines using the .NET SDK and its tools on a build server. The .NET toolset works both interactively, where a developer types commands at a command prompt, and automatically, where a Continuous Integration (CI) server runs a build script. The commands, options, inputs, and outputs are the same, and the only things you supply are a way to acquire the tooling and a system to build your app. This document focuses on scenarios of tool acquisition for CI with recommendations on how to design and structure your build scripts.
+# Use the .NET SDK with Continuous Integration (CI)
```
Comment (IEvangelist) [inline docs/devops/dotnet-cli-and-continuous-integration.md:7]:
```suggestion
# Use the .NET SDK in Continuous Integration (CI) environments
```

**Pull request merged**